java/jooby/config.yaml

```diff
@@ -10,5 +10,5 @@ binaries:
 
 command: >
   /usr/bin/java -server
-  -XX:+UseNUMA -XX:+UseParallelGC -XX:-UseBiasedLocking -XX:+UseStringDeduplication
+  -Xms2g -Xmx2g -XX:+UseNUMA -XX:+UseParallelGC
```

Comment:
out of scope of this PR, but will be nice to homogenize options along jvm based frameworks